### PR TITLE
=doc #24688 link to akka cluster bootstrap in docs

### PR DESCRIPTION
**Pull request by ktoso**
Resolves https://github.com/akka/akka/issues/24688

rather minimalistically but perhaps good

### CODE REVIEW

**Review by johanandren — APPROVED** [review]

File: akka-docs/src/main/paradox/cluster-usage.md [hunk 1]
```diff
@@ -115,10 +121,10 @@ This can also be defined as Java system properties when starting the JVM using t
 Such configuration is typically created dynamically by external tools, see for example:
 
 * [Deploying clustered Akka applications on Kubernetes](http://developer.lightbend.com/guides/k8-akka-cluster/)
```
Comment (patriknw) [inline akka-docs/src/main/paradox/cluster-usage.md]: isn't that guide causing more confusion than helping?
I think we should only link to Bootstrap, and then this doesn't have to be a bullet list.

**Review by patriknw — COMMENTED** [review]
Thanks for picking up this quickly

File: akka-docs/src/main/paradox/cluster-usage.md (hunk 1 above)
Comment (ktoso) [reply akka-docs/src/main/paradox/cluster-usage.md]: Yes great point, will remove

**Comment by ktoso** [comment]
Fixed wording, removed old link.

Please review and merge

**Review by chbatey — APPROVED** [review]
LGTM

**Comment by ktoso** [comment]
Thanks

**Pull request merged**